wiki/Guides/BSS_issues/en.md

```diff
@@ -4,35 +4,39 @@ tags:
   - beatmap submission system issues
 ---
 
+::: Infobox
+![Screenshot of the warning message](img/errormessage.png "The warning message as shown in the editor")
+:::
+
 # How to resolve BSS issues
 
-If the **[Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission)** (***BSS***) won't let you submit your own beatmap because it says that you do not own the map then do the following:
+When submitting a beatmap, the **[Beatmap Submission System](/wiki/Beatmapping/Beatmap_submission)** (***BSS***) may occasionally prevent you from uploading your beatmap by warning that `This beatmap you're trying to submit isn't yours!`. This is  often caused by an incorrectly set parameter in the [.osu file](/wiki/Client/File_formats/osu_(file_format)) or a bugged submission cache, and can be resolved by following these steps:
```

Comment:
Bit confused if this is an accurate statement given the name of the guide. There are multiple ways the upload can fail. The warning `This beatmap you're trying to submit isn't yours!` is but one of them. Therefore, having the guide be called `How to resolve BSS issues` in plural would then not be precise.

Considering this issue is quite common, we can have a section for it, which is what it is being done right below (although it's mixing different troubleshooting steps. May want to split those). Would just make it a little bit more clear, however. Having it in the intro would be misleading.